cmake/alpakaCommon.cmake

```diff
@@ -505,9 +505,14 @@ endif()
 # Find HIP.
 if(ALPAKA_ACC_GPU_HIP_ENABLE)
 
-    # minimal supported HIP version
+    # supported HIP version range
     set(_ALPAKA_HIP_MIN_VER 4.0)
-    find_package(hip "${_ALPAKA_HIP_MIN_VER}")
+    set(_ALPAKA_HIP_MAX_VER 5.0)
+    find_package(hip "${_ALPAKA_HIP_MAX_VER}")
```

Comment:
CMake will not accept ROCm 5.0 if we are searching for 4.0. So we have to search for both versions here.

Reply:
does CMAke care about the minor versions, _e.g._ 5.0.2 vs 5.0 ?

Reply:
It is not actually CMake itself complaining but the CMake configuration script which is part of the ROCm installation. In this case the script (`hip-config-version.cmake`) only cares about major versions, so every 5.x version should work.

Reply:
@steindev observed the same issue with ROCm 5.1 on crusher (ORNL) . I was on the way to open an issue and now saw that we solved the issue already :tada: